zkevm-circuits/src/util/word.rs

```diff
@@ -75,6 +78,12 @@ impl<F: Field> Expr<F> for Word32Cell<F> {
     }
 }
 
+/// Get word from Lo
+pub trait FromLo<T: Default> {
+    /// from lo generic type T unchecked
+    fn from_lo_unchecked(lo: T) -> Word<T>;
+}
```

Comment:
What is the intended use case for this trait?

This trait is more useful if we can implement it for `Field` and `Value<Field>`. But using that trait for  `Field` and `Value<Field>` to implicitly convert unchecked field numbers seems insecure.

I propose we should have a constructor in `Word` to construct the word from lo limb.

Reply:
Right, originally I didn't put into constructors is  because it will conflict the `Word<Expression<F>> ` for `Expression` do not derive `Default` trait. But finally i figure out even define this trait will still conflict with `Expression`. 

Remove this trait entirely!